style.css

```diff
@@ -1 +1,218 @@
 /*  Add your styling here */
+
+:root {
+  --light-pink: rgb(247, 229, 239);
+  --light-orange: rgb(243, 239, 237);
+  ;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-auto-rows: minmax(200px, auto); 
+  grid-gap: 1rem;
+}
+
+header {
+  grid-column: 1 / -1;
+  display: flex;
+  justify-content: space-between;
+  background-color: var(--light-orange);
+}
+
+.company-logo-img {
+  width: 140px;
```

Comment:
remember what was discussed in class about sizing images with **either** 100% height or 100% width and setting a fixed width & height on the parent container, to ensure the aspect ratio of the img stays the same. This image is a square anyway, so it isn't distorted :)